packages/nocodb/src/lib/models/Model.ts

```diff
@@ -175,8 +179,14 @@ export default class Model implements TableType {
         }
       );
 
+      parseMetaProp(modelList);
```

Comment:
Nitpick: Since this function is not defined in the class and because of the name, seems like a  pure function even tho it is change state

Reply:
I can implement a base class and move to that or I will return the value without mutating the object